lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/ExplanationResultGrouper.java

```diff
@@ -0,0 +1,50 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
+
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ExplanationResultGrouper {
```

Comment:
Comment to explain why;

Also, kind of annoying this only works for APExplanations